plugins/bufbuild/connect-kotlin/v0.1.5/buf.plugin.yaml

```diff
@@ -0,0 +1,24 @@
+version: v1
+name: buf.build/bufbuild/connect-kotlin
+plugin_version: v0.1.5
+source_url: https://github.com/bufbuild/connect-kotlin
+description: Idiomatic gRPC & Connect RPCs for Kotlin.
+deps:
+  - plugin: buf.build/protocolbuffers/kotlin:v22.3
+output_languages:
+  - kotlin
+spdx_license_id: Apache-2.0
+license_url: https://github.com/bufbuild/connect-kotlin/blob/v0.1.5/LICENSE
+registry:
+  maven:
+    compiler:
+      kotlin:
+        version: 1.8.20
```

Comment:
Not sure how often we should update this but saw it was recently released: https://github.com/JetBrains/kotlin/releases/tag/v1.8.20